packages/auth/package.json

```diff
@@ -21,6 +21,7 @@
     "@koa/router": "^12.0.0",
     "ajv": "^8.11.0",
     "axios": "^0.27.2",
+    "httpbis-digest-headers": "github:interledger/httpbis-digest-headers",
```

Comment:
how does this install work?

Reply:
https://pnpm.io/cli/add#install-from-git-repository
I don't think https://github.com/interledger/httpbis-digest-headers has been published to npm yet